### DISCUSSION TITLE
More customizable stat counters

### DISCUSSION THREAD

**Discussion by mcendu** [ppy/osu#28413] [Ideas]
Currently most stat counters are only available in the Triangles style, with some counters also in the Argon style. I think these two styles does not blend into every possible skin design.

We can have a new skin element, or a few new skin elements, the only content being a number representing a stat. The skinner uses a dropdown to select a stat the element tracks, like miss count, keys per second, etc. This allows better integration of stat counters with a skin.

Possible styles:
  - Torus (like the Argon key counter)
  - Venera (the current counters, without labels)
  - Argon digits
  - Legacy (in that its skinning and configuration is similar to the legacy score display)

**Comment by BlockHead007** [comment]
#24996?

**Comment by Stoppedpuma** [comment]
In terms of changing fonts I opened a discussion at #27515, more in-depth skinning will happen in the future and you can probably already find some discussions with what you have in mind by searching for what you're wanting implemented.
